src/clients/oidc-react.ts

```diff
@@ -244,23 +233,19 @@ export function createOidcClient(): Client {
           reject(e);
         });
     });
-  };
 
-  const getUserProfile: Client['getUserProfile'] = () => {
-    return getStoredUser();
-  };
+  const getUserProfile: Client['getUserProfile'] = () => getStoredUser();
 
   const getApiAccessToken: Client['getApiAccessToken'] = async options => {
     const user = getStoredUser();
     if (!user) {
       throw new Error('getApiAccessToken: no user with access token');
     }
-    const tokenResponse = await fetchApiToken({
+    return fetchApiToken({
       uri: getTokenUri(getClientConfig()),
       accessToken: user.access_token as string,
       ...options
     });
-    return tokenResponse;
```

Comment:
This doesn't contain an `await` anymore. Is that alright?

Reply:
Same as above